Tests/powershell-yaml.Tests.ps1

```diff
@@ -94,6 +94,23 @@ InModuleScope $moduleName {
         }
 
         Context "Test array handling under various circumstances." {
+
+            It "Should ConvertTo empty array." {
+                $str = ConvertTo-Yaml @()
+                $str.Trim() | Should -Be "[]"
+            }
+
+            It "Should ConvertTo empty array with -OutFile." {
+                $tempFile = [System.IO.Path]::GetTempFileName()
+                ConvertTo-Yaml @() -OutFile $tempFile -Force
+                (Get-Content $tempFile).Trim() | Should -Be "[]"
+            }
+
+            It "Should ConvertFrom empty array." {
+                $arr = "[]" | ConvertFrom-Yaml
+                Confirm-Equality ([object[]] $arr) @() | Should -Be $true
```

Comment:
I think you can just check if it is ```$null``` which it should be

Reply:
I am confused now. I was expecting that `[]` should turn into a non-null array with `.Length -eq 0`.

Am I missing something?